mitosheet/src/components/Mito.tsx

```diff
@@ -860,23 +860,23 @@ export const Mito = (props: MitoProps): JSX.Element => {
 
     // Check which taskpanes are open
     const taskpaneOpen = uiState.currOpenTaskpane.type !== TaskpaneType.NONE;
-    const graphTaskpaneOpen = uiState.currOpenTaskpane.type === TaskpaneType.GRAPH && uiState.selectedTabType === 'graph';
-    const narrowTaskpaneOpen = taskpaneOpen && !graphTaskpaneOpen;
+    const wideTaskpaneOpen = uiState.currOpenTaskpane.type === TaskpaneType.GRAPH && uiState.selectedTabType === 'graph';
```

Comment:
I think this name might be worse, but wideTaskpane just makes it sound like it's a taskpane but a bit wider, where in actuality, the graphTaskpane / wideTaskpane is actually a full sheet! Which is horrible, but it is.